acceptancetests/utility.py

```diff
@@ -322,10 +322,17 @@ def add_basic_testing_arguments(
 add_basic_testing_arguments.__test__ = False
 
 
-def configure_logging(log_level):
+def configure_logging(log_level, logger=None):
```

Comment:
This change enables us to alter specific loggers, which are used within some of the `assess_*.py` files.